test/exe/main_common_test_base.h

```diff
@@ -0,0 +1,77 @@
+#pragma once
```

Comment:
note to reviewers: this was TestBase class was factored out from main_common_test.cc to make it easier to split off AdminResponseTest into a separate file.